scripts/tekton/openshift.yaml

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
Ditto re. Kabanero / collections

Reply:
Removed "collections"
What do you suggest as alternative for "kabanero"

Reply:
Done